code/modules/antagonists/_common/antag_datum.dm

```diff
@@ -34,6 +34,8 @@ GLOBAL_LIST_EMPTY(antagonists)
 	var/can_elimination_hijack = ELIMINATION_NEUTRAL
 	///If above 0, this is the multiplier for the speed at which we hijack the shuttle. Do not directly read, use hijack_speed().
 	var/hijack_speed = 0
+	/// This is the place we take the shuttle if we hijack it. See defines for what
```

Comment:
Don't just say "defines", there's thousands of them. Say what defines are actually valid here.